phongo_compat.h

```diff
@@ -43,28 +43,6 @@
 # include <Zend/zend_string.h>
 #endif
 
-#if PHP_VERSION_ID >= 50500
-#define ITERATOR_GET_CURRENT_KEY(it, z) \
-		(it).funcs->get_current_key(&(it), z TSRMLS_CC);
-#else
-#define ITERATOR_GET_CURRENT_KEY(it, z) \
-{ \
-	char *str_key; \
-	uint str_key_len; \
-	ulong int_key; \
-	switch ((it).funcs->get_current_key(&(it), &str_key, &str_key_len, &int_key TSRMLS_CC)) { \
-		case HASH_KEY_IS_LONG: \
-			ZVAL_LONG(z, int_key); \
-			break; \
-		case HASH_KEY_IS_STRING: \
-			ZVAL_STRINGL(z, str_key, str_key_len-1, 0); \
-			break; \
-		default: \
-			RETURN_NULL(); \
-	} \
-}
-#endif
-
```

Comment:
Was this define not used at all? Don't see it referenced anywhere else in this PR?